web/cobrands/fixmystreet/map.js

```diff
@@ -18,6 +18,12 @@ var fixmystreet = fixmystreet || {};
         }
     });
 
+    // bodies data needs slightly different handling
+    if (map_data.getAttribute('data-bodies')) {
+       var bodies = '' + map_data.getAttribute('data-bodies');
```

Comment:
The attribute will always be a string, so no need for `'' + ` here.

Reply:
In fact, looks like you could add `bodies` to the `map_key` array and not have this bit at all, replacing it with a line like the `fixmystreet.areas` one underneath.